src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -140,4 +148,19 @@ public boolean equals(Object other) {
                 || (other instanceof UniquePersonList // instanceof handles nulls
                         && this.internalList.equals(((UniquePersonList) other).internalList));
     }
+
+
+    /*Comparator for sorting the list by Student Name*/
+    public static Comparator<Person> PersonNameComparator = new Comparator<Person>() {
```

Comment:
Good that you have added comment explaining the role of the comparator. However, I would suggest to additionally mention that it is case-insensitive since is not trivial